modules/integration_azure-firewall/conf/02-throughput.yaml

```diff
@@ -0,0 +1,25 @@
+## Example
+module: Azure firewall
+name: throughput
+
+transformation: true
+aggregation: true
+
+filtering: "filter('resource_type', 'Microsoft.Network/azureFirewalls') and filter('primary_aggregation_type', 'true')"
+
+tip: "Firewall will autoscale at 2.5Gbps but the maximum throughput is 30Gbps"
+
+signals:
+  throughput:
+    metric: "Throughput"
+  signal:
+    formula: "throughput.scale(0.000000953674316)"
+rules:
+  critical:
+    threshold: 25600
+    comparator: ">"
+    lasting_duration: "900s"
```

Comment:
a `min(over='15m')` transformation will basically do the same but you will can use dependency between rule without problem.

I left this comment for azure reviewers ;)